bodhi/client/bindings.py

```diff
@@ -71,6 +71,9 @@ def __init__(self, update):
         """Initialize the Exception."""
         self.update = six.text_type(update)
 
+    def __str__(self):
```

Comment:
The failure might be due to @frenzymadness having added the same name in his patch, so now it's defined twice.